classes/game.rb

```diff
@@ -0,0 +1,27 @@
+require 'date'
+require_relative './item'
+# create game class
+class Game < Item
+  attr_accessor :name, :multiplayer, :last_played_at, :published_date
+
+  def initialize(name, multiplayer, last_played_at)
+    super(archived, published_date)
+    @name = name
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
```

Comment:
- [ ] When I try to add a new game It gives me the error in the image below and that's because you give 5 arguments in `add_game` method and here you only give 3 parameters to the `initialize `method, So please fix that.

![image](https://user-images.githubusercontent.com/68796232/187040744-3ff7566b-5402-425c-a88e-91ca36f714c3.png)